e2e/autoresizer-cluster.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion:  kind.x-k8s.io/v1alpha4
+kind: Cluster
+# patch the generated kubeadm config with some extra settings
+kubeadmConfigPatches:
+- |
+  apiVersion: "kubeadm.k8s.io/v1beta2"
+  kind: ClusterConfiguration
+  metadata:
+    name: config
+  kubernetesVersion: "v@KUBERNETES_VERSION@"
+  networking:
+    serviceSubnet: 10.0.0.0/16
+  scheduler:
+    extraVolumes:
+      - name: "config"
+        hostPath: /mnt/host/scheduler
+        mountPath: /var/lib/scheduler
+        readOnly: true
+    extraArgs:
+      config: /var/lib/scheduler/scheduler-config.yaml
+- |
+  apiVersion: "kubeadm.k8s.io/v1beta2"
+  kind: JoinConfiguration
+  metadata:
+    name: config
+  nodeRegistration:
+    kubeletExtraArgs:
+      volume-stats-agg-period: 1s
```

Comment:
Could you write comments about the difference between this file with TopoLVM one to maintain it?

Reply:
I added comment.
https://github.com/topolvm/pvc-autoresizer/pull/44/commits/cc119abb1acd783869e930e4abd5162cd0e110b3